README.md

```diff
@@ -31,16 +31,19 @@ $ ./setup
 
 #### Running
 
+To run Rosie, you need to select a module to be called.
+For example, if you want to run `chamber_of_deputies` module, you should run this command:
```

Comment:
Rosie still runs without calling a submodule doesn't she? If so I would add the information that you can call upon a submodule or you can run all of them too if you like. What do you think?